install_local.sh

```diff
@@ -19,7 +19,10 @@ if ! command -v gdown &> /dev/null; then
     exit 1
 fi
 
-CRT_DIR=$(pwd)
+CRT_DIR=$(pwd)                                 # .../intball2_ws/src/int-ball2_isaac_sim
```

Comment:
This should work well.
What about calling it `PKG_DIR` for better readability?
```suggestion
PKG_DIR=$(pwd)                                 # .../intball2_ws/src/int-ball2_isaac_sim
```